packages/frontend/src/pages/RestoreAccount/RestoreAccount.tsx

```diff
@@ -0,0 +1,131 @@
+import { removeAllAccounts } from './lib/accounts';
+import {
+    Title,
+    ConnectedWallets,
+    Description,
+    UserItem,
+    Username,
+    UserContent,
+    Label,
+    CheckboxCaption,
+    CheckboxWrapper,
+    Buttons
+} from './ui';
+import CheckIcon from './ui/CheckIcon';
+
+type RestoreAccountProps = {
+    availableAccounts: string[];
+}
+
+const RestoreAccount: FC<RestoreAccountProps> = ({
+    availableAccounts
+}) => {
+    const { t } = useTranslation();
+    const [checked, setChecked] = useState(false);
+    const [redirectUrl, setRedirectUrl] = useState('');
+
+    const handleRestore = useCallback(async () => {
+        await removeAllAccounts(availableAccounts);
+        setRedirectUrl('/recover-seed-phrase');
+    }, [availableAccounts]);
+
+    if (redirectUrl.length > 0) {
+        return (
+            <Redirect
+                to={{
+                    pathname: redirectUrl,
+                }}
+            />
+        );
+    }
+
+    return (
+        <Container className='small-centered border'>
+            <Title>
+                {t('restoreAccount.title')}
+            </Title>
+            <ConnectedWallets>
+                {t('restoreAccount.wallets', {
+                    count: availableAccounts.length
+                })}
+            </ConnectedWallets>
+            <Description>
+                {t('restoreAccount.description')}
+            </Description>
+
+            <div>
+                {
+                    availableAccounts.map((account) => (
+                        <UserItem key={account}>
+                            <UserContent>
+                                {(
+                                    /*@ts-ignore*/
+                                    <UserIcon>
+                                        <UserIconGrey color='#9a9a9a' />
+                                    </UserIcon>
+                                )}
+                                <Username>
+                                    {account}
+                                </Username>
+                            </UserContent>
+                            {checked && (
+                                <CheckIcon />
+                            )}
+                        </UserItem>
+                    ))
+                }
+            </div>
+            <CheckboxWrapper>
+                <Label>
+                    {(
+                        /*@ts-ignore*/
+                        <Checkbox
+                            checked={checked}
+                            onChange={currentTargetChecked(setChecked)} />
+                    )}
+                    <CheckboxCaption>
+                        {t('restoreAccount.checkboxCaption')}
+                    </CheckboxCaption>
+                </Label>
+            </CheckboxWrapper>
+            <Buttons>
+                {(
+                    <FormButton
+                        /*@ts-ignore*/
+                        onClick={console.log}
```

Comment:
Why is it `console`? We need to show "_enter password_" form, don't we?

Reply:
fixed